### PR TITLE
Update language to make the labeling needs more clear

### PR DESCRIPTION
**Pull request by ebenoist**
Turns out you can't read that the variable's name is label once its been interpolated :)

### CODE REVIEW

File: src/check_for_review.ts [hunk 1]
```diff
@@ -12,12 +12,15 @@ const {
   posthocApprovalLabel,
 } = getInput();
 
+const MISSING_MESSAGE = `
+This issue is missing verification by a peer!
+Have a peer apply the ${posthocApprovalLabel} label once they've reviewed your emergency change.
```
Comment (stevenweber) [inline src/check_for_review.ts:17]:
```suggestion
When a PR is merged after bypassing approval as this PR did, a peer needs to review these changes afterwards and add the ${posthocApprovalLabel} label to document that this code was eventually reviewed.
```

Comment (stevenweber) [reply src/check_for_review.ts:17]: Or is this message used for both prs and issues?

**Comment by ebenoist** [comment]
Updated. Thanks!

**Review by stevenweber — APPROVED** [review]

File: src/check_for_review.ts (hunk 1 above)
Comment (ebenoist) [reply src/check_for_review.ts]: Ugh yeah, it is, which is why I was originally vague. I'll change the language around a bit more.

**Review by stevenweber — APPROVED** [review]

**Comment by ebenoist** [comment]
Updated with some issue/pr compatible language

**Pull request merged**